src/adapter/src/coord.rs

```diff
@@ -1595,6 +1595,8 @@ impl Coordinator {
         debug!("coordinator init: installing existing objects in catalog");
         let mut privatelink_connections = BTreeMap::new();
 
+        let local_read_ts_for_index_bootstrapping = self.get_local_read_ts().await;
+
```

Comment:
Btw. there is another `get_local_read_ts()` call later in `bootstrap()`, but that is too late for me, and I was not sure whether that could be moved here. It probably can't simply be moved here, right?

Reply:
I think duplicating the call here is correct. The later call to `get_local_read_ts` comes after a call to `get_local_write_ts`. So if we were to generate a read ts here and then use that read ts later on, I think the guarantees about the read/write ts relationships would be lost.